server.py

```diff
@@ -14,18 +14,15 @@
 from db import Code, LoginCode, PendingEmail, User
 import util
 
+config = util.loadYaml('config.yaml')
 
 CODE_VALIDATOR = re.compile(r'^(\w{8})$')
 CODE_SIZE = 8
 LOGIN_COOKIE_SIZE = 16
 
-NAME = 'Funbox'
```

Comment:
We can keep this constant, but set it to the config field value instead. That will cut down on the number of times we need to repeat `config['server_name']`